test/execution/test_plan.py

```diff
@@ -640,6 +640,7 @@ def forward(self, x):
     # Compare with local plan
     assert th.all(decrypted - expected.detach() < 1e-2)
     # assert fetched_plan.state.state_placeholders != plan.state.state_placeholders #TODO
```

Comment:
Q: Are the state placeholder still not working?

Reply:
What's the problem with state placeholders?

Reply:
(I see that #3214 has something to do with `Parameters`, but not sure what about `Placeholders` is problematic.)